vendor/bat-native-ledger/src/bat/ledger/internal/response/response_api.cc

```diff
@@ -103,6 +103,9 @@ ledger::Result ParseParameters(
   }
 
   for (const auto& choice : ac_choices->GetList()) {
+    if (!choice.is_bool()) {
```

Comment:
Why do we check for bool, when we end up pulling out a double?

Reply:
nice catch, stupid copy/paste 😄